README.md

```diff
@@ -40,13 +40,13 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 
 ## Posts
 
-(Persistent Storage API: Building for the offline web)[https://deanhume.com/Home/BlogPost/persistent-storage-api--building-for-the-offline-web/10161]
+[Persistent Storage API: Building for the offline web])https://deanhume.com/Home/BlogPost/persistent-storage-api--building-for-the-offline-web/10161)
```

Comment:
It's incorrect markdown. Please fix: `])` -> `](`.
```
[Persistent Storage API: Building for the offline web](https://deanhume.com/Home/BlogPost/persistent-storage-api--building-for-the-offline-web/10161)
```

Reply:
oooh snap.

Reply:
@gurza there, I had to do a proper clone and fixed more of them.